optashift-employee-rostering-webapp/pom.xml

```diff
@@ -225,28 +225,8 @@
         </property>
```

Comment:
Should we add another activation here for cargo.maven.skip?

Reply:
(It just above this line, but GitHub won't let me comment on it)

Reply:
The maven-cargo-plugin directly supports the environment variable "cargo.maven.skip". There's no need to add an activation for that.

The goal if this code is to make sure that "-DskipTests" also skips cargo. Ideally, the maven-cargo-plugin should also pick up the "skipTests", but it doesn't.
There's no need to do any activation when a user only wants to skip cargo.